doc/source/image_sequence.rst

```diff
@@ -14,6 +14,11 @@ filenames.
 * the filepath of a zipped archive, such as ``'my_directory/all-images.zip'``
 * a list of filepaths, such as ``['image1.png', 'image2.png']``
 
+In all cases except the last one, to sort the filenames, sorting keys are
```

Comment:
This is clear but has a typo ("are generated are sorted"). Would it also help to give a short example? You could say something like

```
['abc3.png', 'abc12.png', 'def3.png'] are in sorted order
```

Reply:
Sure, fixed & included example.